grisera_api/experiment/experiment_service_ontology.py

```diff
@@ -12,9 +15,39 @@ class ExperimentServiceOntology(ExperimentService):
     """
     ontology_api_service = OntologyApiService()
 
-    def get_experiment(self, experiment_id):
-        pass
+    def get_experiment(self, experiment_label: Union[int, str], depth: int = 0):
+        """
+        Send request to ontology api to get given experiment
+        Args:
+            experiment_label (int | str): label of experiment
+            depth (int) : only for compatibility with graph_api, always set to 0
+        Returns:
+            Result of request as experiment object
+        """
+        model_id = 1
```

Comment:
Why is `model_id` explicitly set to 1?

Reply:
@mrwrob we answered this question in this pull request:
https://github.com/GRISERA/grisera-framework/pull/198

Reply:
👍